factory/src/main/java/com/agileactors/factory/v3/api/TransportationController.java

```diff
@@ -0,0 +1,31 @@
+package com.agileactors.factory.v3.api;
+
+import com.agileactors.factory.v3.dtos.TransportationDto;
+import com.agileactors.factory.v3.dtos.TransportationInput;
+import com.agileactors.factory.v3.mappers.TransportationMapper;
+import com.agileactors.factory.v3.services.transportation.TransportationService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/api/v2/transportation")
+public class TransportationController {
+
+  private TransportationService transportationService;
```

Comment:
I suggest you use private final members with @RequireArgsConstructor since that way you declare that the private member is mandatory for the controller to work correctly. If possible apply this to other similar cases